python/risk_meters/historical_vuln_count.py

```diff
@@ -0,0 +1,187 @@
+        print(f"Reports don't have matching IDs: {vuln_count_report['id']}, {accepted_risk_resp['id']}")
+        sys.exit(1)
+    
+    if vuln_count_resp['id'] != id:
+        print(f"Reports don't matching requested ID: {vuln_count_report['id']}, {id}")
+        sys.exit(1)
+
+    # Get to data as a dictionary of dates with each element containing low, medium, and high.
+    vuln_count_data = vuln_count_resp['historical_vulnerability_count_by_risk']
+    risk_accepted_data = accepted_risk_resp['risk_accepted_over_time']
+
+    # Initialize output table.
+    vuln_count_tbl = PrettyTable()
+    vuln_count_tbl.field_names = ["Date", "High", "RA High", "Medium", "RA Medium", "Low", "RA Low"]
+
+    # Process both vulnerability count and risk accepted vulnerability count data.
+    for (vc_date, ra_date) in zip(vuln_count_data, risk_accepted_data):
+        if vc_date != ra_date:
+            print(f"Dates don't match: {vc_date} : {ra_date}")
+            sys.exit(1)
+
+        # Add today or the first of the month to the table.
+        if one_shot or vc_date[-2:] == "01":
+            vc_high = int(vuln_count_data[vc_date]['high'])
+            vc_medium = int(vuln_count_data[vc_date]['medium'])
+            vc_low = int(vuln_count_data[vc_date]['low'])
+            ar_high = int(risk_accepted_data[ra_date]['high'])
+            ar_medium = int(risk_accepted_data[ra_date]['medium'])
+            ar_low = int(risk_accepted_data[ra_date]['low'])
+   
+            vuln_count_tbl.add_row([vc_date, vc_high, ar_high, vc_medium, ar_medium, vc_low, ar_low])
+        
+    print(vuln_count_tbl)
+    print("")
+
+def print_help(program):
+    print(f"Prints out the historical risk meter vulnerability counts for a year.")
+    print(f"{program} <risk meter name> [today]")
+    print(f"   <risk meter name> - name of the risk meter")
+    print(f"   [today] - if present will only print the vulnerability counts for today")
+    print("")
+
+if __name__ == "__main__":
+    # Set the one_shot flag to False.
+    one_shot = False
+    check_risk_meter_name = True
+
+    # Look for the risk meter name.
+    if len(sys.argv) > 1:
+        risk_meter_name = sys.argv[1]
+    else:
+        print("Requires risk meter name.")
+        print_help(sys.argv[0])
+        sys.exit(1)
+
+    # Look for the "today" keyword which tells this program to do only
+    # today and not monthly for a year.
+    if len(sys.argv) > 2:
+        if sys.argv[2] == "today":
+            one_shot = True
+        else:
+            print("Currently only 'today' is supported.")
+            print_help(sys.argv[0])
+            sys.exit(1)
+
+    # Check for arguments that could show if run as a subprocess.
+    # If the paramters are invalid, ignore them and proceed.
+    # If the parameters are valid, set the check_risk_meter_name to False.  This
+    # reduces the number of API invokes.
+    if len(sys.argv) > 3 and len(sys.argv) < 7:
```

Comment:
Optional: You could potentially hard bail-out here if len(sys.argv) > 6, since more than 6 args is invalid.